src/main/java/teammates/common/util/Const.java

```diff
@@ -953,6 +953,9 @@ private Const() {
         public static final String INSTRUCTOR_COURSE_STUDENT_DELETE_ALL = "/page/instructorCourseStudentDeleteAll";
         public static final String INSTRUCTOR_COURSE_STUDENT_LIST_DOWNLOAD = "/page/instructorCourseStudentListDownload";
         public static final String INSTRUCTOR_COURSE_ENROLL_PAGE = "/page/instructorCourseEnrollPage";
+        public static final String INSTRUCTOR_COURSE_ENROLL_AJAX_ENROLL_STATUS_PAGE =
+                "/page/instructorCourseEnrollAjaxEnrollStatusPage";
+        public static final String INSTRUCTOR_COURSE_ENROLL_AJAX_PAGE = "/page/instructorCourseEnrollAjaxPage";
```

Comment:
Might need to change the name of this action given that there might be more AJAX actions in the enroll page. 

Note to look into this after the current code behavior is acceptable.